sale_automatic_workflow/models/sale_order.py

```diff
@@ -44,8 +55,6 @@ def _onchange_workflow_process_id(self):
         if not self.workflow_process_id:
```

Comment:
@trisdoan @sebalix Maybe it's time to change the field to computed/stored/readonly=False one to remove onchanges.

Reply:
Hi @rousseldenis @sebalix , changes done. Please help to review :pray:

Reply:
Hello @rousseldenis @sebalix, I updated as you suggest

I added this [commit](https://github.com/OCA/sale-workflow/pull/3105/commits/028edd74b2aafc33b7a8c6f9133242fa5c79ed30).Adapt field `picking_policy` in sale.order to writable computed field. Unfortunately, there is a module: `sale_order_type`, which does the same thing but depends on field 'type_id' instead.
-> Mark sale_order_type as rebel module

Reply:
Not sure it's a good idea :thinking: 
What you could do is to extract the `@api.depends` values in a method and make use of `hasattr` like it's done in the compute method:
```python
# in sale_order_type
def _depends_picking_policy(self):
    depends = []
    if hasattr(super(), "_depends_picking_policy"):
            res = super()._depends_picking_policy()
    depends.append("type_id")
    return depends

@api.depends(lambda self: self._depends_picking_policy())
def _compute_picking_policy():
    [...]
```
```python
# in sale_automatic_workflow_stock
def _depends_picking_policy(self):
    depends = []
    if hasattr(super(), "_depends_picking_policy"):
            res = super()._depends_picking_policy()
    depends.append("workflow_process_id")
    return depends

@api.depends(lambda self: self._depends_picking_policy())
def _compute_picking_policy():
    [...]
```
This way we are dynamically compatible with both module installed (and probably still with an undefined behavior, but that's better than something else I guess).

Reply:
pr is coming today. Will ping when it's done

Reply:
Hello @sebalix, updated as you suggest + tests are green :pray: